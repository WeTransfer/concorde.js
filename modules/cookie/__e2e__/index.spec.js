const puppeteer = require('puppeteer');

describe('Cookie bundle', async () => {
  let browser;

  beforeAll(async () => {
    browser = await puppeteer.launch({ args: ['--no-sandbox', '--disable-setuid-sandbox'] });
  });

  afterAll(() => {
    browser.close();
  });

  it('should render browser info', async () => {
    const page = await browser.newPage();
    await page.goto('http://localhost:8080/modules/cookie/__e2e__/index.html');
    await page.waitForSelector('#results');

    const results = await page.$eval('#results', (e) => e.innerHTML);
    expect(results).toMatchSnapshot();
  });
});
