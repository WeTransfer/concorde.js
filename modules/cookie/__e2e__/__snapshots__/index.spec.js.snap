// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cookie bundle should render browser info 1`] = `
"
    <p><strong>Default value:</strong> <span id=\\"default_value\\">default</span></p>
    <p><strong>Set/Get value:</strong> <span id=\\"set_get_value\\">foo</span></p>
    <p><strong>Unset value:</strong> <span id=\\"unset_value\\"></span></p>
  "
`;
